plugin/fzf.vim

```diff
@@ -484,7 +486,9 @@ try
     elseif type == 3
       let temps.input = s:fzf_tempname()
       call s:writefile(source, temps.input)
-      let source_command = (s:is_win ? 'type ' : 'cat ').fzf#shellescape(temps.input)
+      " Disable shell escape for git bash, as it breaks the command here
+      let source_command = (s:is_win_cmd ? 'type ' : 'cat ')
+        \.(!s:is_win || !exists('$SHELL') ? fzf#shellescape(temps.input) : substitute(temps.input, '\', '/', 'g'))
```

Comment:
Shouldn't we still `shellescape` (not `fzf#shellescape`) `temps.input` for safety?

Reply:
The expression is getting a bit too complicated. How about splitting it into if-else statements?
```sh

if !s:is_win
  let source_command = ...
elseif exists('$SHELL')
  " Git bash
  let source_command = ...
else
  " cmd.exe
  let source_command = ...
end
```